controllers/grid/queries/traits/StageMailable.php

```diff
@@ -30,7 +31,7 @@ abstract public function getStageId();
     /**
      * @return Mailable which corresponds to the given workflow stage
      */
-    protected function getStageMailable(Context $context, Submission $submission): Mailable
+    protected function getStageMailable(Context $context, Submission $submission, string $subject = "", string $message = ""): Mailable
```

Comment:
Use single quotes, `'`, instead of double quotes, `"`, to define the strings.

Reply:
ok